src/models/flashcard.rs

```diff
@@ -19,9 +19,15 @@ impl FlashCard {
         }
     }
 
+
     pub fn flip_card(&mut self) {
         self.flipped = !self.flipped;
     }
+
+    pub fn instructions() -> String {
+        return String::from(r#"SPACE: Show cards back"#)
```

Comment:
Using a &str here is fine

```suggestion
        return String::from("SPACE: Show cards back")
```